tabbycat/notifications/views.py

```diff
@@ -117,8 +117,11 @@ def get_tables(self):
 class EmailEventWebhookView(TournamentMixin, View):
 
     def post(self, request, *args, **kwargs):
-        if kwargs['key'] is not self.tournament.pref('email_hook_key'):
-            return HttpResponse(status=404) # 404: Not Found
+        if not self.tournament.pref('email_hook_key'):
+            return HttpResponse(status=403) # 403: Forbidden
+
+        if kwargs['key'] != self.tournament.pref('email_hook_key'):
```

Comment:
@tienne-B heads-up, `!=` and `is not` aren't the same thing, and if you intend to compare values (rather than identities), you should use `!=` rather than `is not`. I needed to change this in order to get the webhook to be recognized by Tabbycat—I'm curious that you didn't find similarly?